a/apache-arrow/build_info.json

```diff
@@ -1,17 +1,17 @@
 {
     "package_name" : "apache-arrow",
     "github_url":"https://github.com/apache/arrow.git",
-    "version":"go/v10.0.1",
+    "version":"go/v12.0.1",
     "default_branch":"main",
     "package_dir":"a/apache-arrow/",
-    "build_script" :"arrow_ubi_8.5.sh",
+    "build_script" :"arrow_ubi_8.7.sh",
     "docker_build": false,
     "validate_build_script": true,
     "use_non_root_user": false,
     
-    "go/v10.0.*" : {
+    "go/v12.0.*" : {
```

Comment:
Here we dont want to remove support for old version. SO better to keep old entry along with new version